examples/cpp_exec/serialized_fn/serialize.py

```diff
@@ -0,0 +1,45 @@
+r"""Sample code to show how to serialize and deserialize a JAX executable.
+
+To invoke:
+  $ bazel run -c opt examples/cpp_exec/serialized_fn:serialize -- /tmp/serialized_fn
+"""
+
+import jax
+from jax._src import xla_bridge as xb
```

Comment:
Also please don't access `_src` in user examples.

To get the backend, you can get it from devices.